internal/jennies/golang/types.go

```diff
@@ -162,6 +164,42 @@ func formatScalar(val any) string {
 	return fmt.Sprintf("%#v", val)
 }
 
+func formatDefaultStruct(refPkg, pkg string, structMap *orderedmap.Map[string, interface{}]) string {
+	starter, format, sep, lastSep, ending := "{\n", "%s: %v", ",\n", ",\n", "}"
+	if pkg != "" {
+		starter, format, sep, lastSep, ending = fmt.Sprintf("New%sBuilder().\n", pkg), "%s(%v)", ".\n", ",\n", ""
```

Comment:
Are we using the package as builder name here? If so, that's probably incorrect.

See https://github.com/grafana/cog/blob/2fd0d68c215e8231b9184d6b2d0d1b05c446fd4c/internal/jennies/golang/templates/builders/builder.tmpl#L18